index.html

```diff
@@ -53,14 +53,11 @@
       <div class="collapse navbar-collapse" id="navbarSupportedContent-555">
         <ul class="navbar-nav mr-auto text-dark">
           <li class="nav-item">
-            <a class="nav-link" href="#">About </a>
+            <a class="nav-link" href="/Users/Vny/BootCamp/Project/project/project-1/team.html"> Team </a>
```

Comment:
Will a local path like this one work on other peoples computers or does this need to be a link to a published website?

Reply:
I agree, Its by mistake, i meant to use relative path

Reply:
Just to record it here, Prasanna is going to fix this on another branch after we merge